packages/components/src/components/Graph/Graph.js

```diff
@@ -59,7 +67,14 @@ export default class Graph extends PureComponent {
   };
 
   render() {
-    const { height, isSubGraph, width, y } = this.props;
+    const {
+      height,
+      isSubGraph,
+      onClickStep,
+      onClickTask,
+      width,
+      y
```

Comment:
this was applied to subgraphs to nudge them down a pixel to avoid overlapping the border on the task label in the selected state, looks like it's not needed anymore after other layout changes so removing